utils/ipfs.py

```diff
@@ -65,6 +101,10 @@ def _upload_str_to_web3_storage(text: str) -> str:
 
 
 def _upload_str_to_ipfs(text: str) -> str:
+    if get_pinata_cloud_token(silent=True):
+        return _upload_str_to_web3_storage(text)
```

Comment:
Should be `_upload_str_to_pinata_cloud` instead of `_upload_str_to_web3_storage`